doc/interop-test-descriptions.md

```diff
@@ -301,12 +279,11 @@ Procedure:
     * call was successful
     * exactly four responses
     * response payloads are COMPRESSABLE
-    * response compression is consistent with the requested one.
-    * if `response_compression == NONE`, the response MUST NOT have the
-      compressed message flag set.
-    * if `response_compression != NONE`, the response MUST have the compressed
-      message flag set.
-    * response payload bodies are sized (in order): 31415, 9, 2653, 58979
+    * if `request_compressed_response` is false, the response's messages MUST
+      NOT have the compressed message flag set.
+    * if `request_compressed_response` is true, the response's messages MUST
+      have the compressed message flag set.
+    * response payload bodies are sized (in order): 31415, 59, 2653, 58979
```

Comment:
actually, Pi is 3.141592653....  so 31415  and 59  are wrong and it should be 31415  and 9.  But is seems like that ALL the other streaming interop tests are wrong in this respect, so making it consistent makes sense.